src/blocks/stars.css

```diff
@@ -0,0 +1,16 @@
+/* #region stars */
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
Avoid using tag names for styling. Instead of using `:first-child` and `:nth-child`, consider applying a class to the elements you want to style. This will improve performance and maintainability.